lib/functions.php

```diff
@@ -551,7 +551,7 @@ function elgg_solr_add_update_object_default($entity) {
 	$doc->enabled = $entity->enabled;
 
 	$params = array('entity' => $entity);
-	$doc = elgg_trigger_plugin_hook('elgg_solr:index', $entity->type, $params, $doc);
+	$doc = elgg_trigger_plugin_hook('elgg_solr:index', $entity->type . (($subtype != '') ? ':' . $subtype : ''), $params, $doc);
```

Comment:
That's true, its a BC change. And, we may add `:all`, so you're able to, for example, hook all objects at once (`object:all`). I thought this is used on more places in Elgg.